bravado/fido_client.py

```diff
@@ -38,6 +38,10 @@ def text(self):
         return self._delegate.body
 
     @property
+    def raw_bytes(self):
+        return self._delegate.body
```

Comment:
This seems to be a limitation of fido - the text property should return unicode, the raw_bytes property bytes. I'm assuming self._delegate.body is bytes, so this should be fine (and we should fix `text` above).